src/middleware/packages/activitypub/services/relay.js

```diff
@@ -0,0 +1,331 @@
+const urlJoin = require('url-join');
+const { MoleculerError } = require('moleculer').Errors;
+const fetch = require('node-fetch');
+const { ACTIVITY_TYPES, ACTOR_TYPES, OBJECT_TYPES, PUBLIC_URI } = require('../constants');
+const { MIME_TYPES } = require('@semapps/mime-types');
+
+const { createFragmentURL, getContainerFromUri, isMirror } = require('@semapps/ldp/utils');
+
+const { defaultToArray } = require('@semapps/activitypub/utils');
+
+const regexPrefix = new RegExp('^@prefix ([\\w-]*: +<.*>) .', 'gm');
```

Comment:
`regexPrefix` n'est pas utilisé. Ni les imports de `createFragmentURL` et `MoleculerError`

Reply:
you are right. copié/collé depuis le service mirror